Mtoff add latency breakdown info (#16264)
DataDog/documentation: content/en/tracing/trace_explorer/visualize.md

The configuration of the columns is stored alongside other elements of your troubleshooting context in saved views.

The `Latency Breakdown` of the trace might be missing for some spans if the trace is malformed or incomplete. For instance, the error and the rare samplers capture pieces of traces, without the guarantee of capturing the complete trace. In this case, the data is omitted to avoid displaying inconsistent of misleading latency information that would only make sense when the trace is complete.

> 

Author: To avoid displaying misleading information, a malformed or incomplete trace will not contain a `Latency Breakdown`. Error and rare samplers may explain some of these cases: they prioritize spans without the guarantee of capturing a complete trace.

Author: @estherk15

> If the trace is somehow malformed (for example if there is a loop in the trace or if the root span is missing) it could result in spans without data under the "LATENCY BREAKDOWN" column. In some cases, a root service may decide to drop a trace while a downstream service decides to keep it. Only the downstream-service spans are ingested in this case, resulting in an incomplete trace. It is misleading to display a latency breakdown for an incomplete trace, so the data is omitted. 
> 

Reviewer: ```suggestion
The `Latency Breakdown` of the trace might be missing for some spans if the trace is malformed or incomplete. For instance, the error and the rare samplers capture pieces of traces, without the guarantee of capturing the complete trace. In this case, the data is omitted to avoid displaying inconsistent of misleading latency information that would only make sense when the trace is complete.

```

> You may see spans without data under the "LATENCY BREAKDOWN" column (There are some examples pictured in the GIF above). This can happen if the trace is somehow malformed, for example if there is a loop in the trace or if the root span is missing. Sometimes, the root service may decide to drop the trace but a downstream service thinks it’s interesting to keep. This means that only the downstream-service spans are ingested, resulting in an incomplete trace. It would be misleading to display a latency breakdown for an incomplete trace, so in these cases, the data is omitted instead. 

Reviewer 2: ```suggestion
If the trace is somehow malformed it could result in spans without data under the "LATENCY BREAKDOWN" column. For example, if there is a loop in the trace or if the root span is missing, the root service decides to drop the trace but a downstream service decides to keep it. This means that only the downstream-service spans are ingested, resulting in an incomplete trace. It is misleading to display a latency breakdown for an incomplete trace, so the data is omitted. 
```
